tests/unit/store/flask.spec.js

```diff
@@ -168,12 +168,12 @@ describe("store/flask", () => {
       spied_set_interval.mockReturnValueOnce(expected_interval_id);
 
       await store.dispatch("waveform/start_get_waveform_pinging");
-      expect(spied_set_interval.mock.calls.length).toBe(1);
-      expect(store.state.waveform.waveform_ping_interval_id).toEqual(
+      expect(spied_set_interval.mock.calls).toHaveLength(1);
+      expect(store.state.waveform.waveform_ping_interval_id).toStrictEqual(
         expected_interval_id
       );
     });
-    test("Given the current state is calibrating and the returned state is CALIBRATED, Then the URL should include the current state UUID and the vuex status state should update to CALIBRATED and the Vuex Playback State should update to CALIBRATED", async () => {
+    test("Given the current state is calibrating, When the returned status returned as CALIBRATED, Then the URL should include the current state UUID and the vuex status  should update to CALIBRATED and the Vuex Playback State should update to CALIBRATED", async () => {
```

Comment:
mocking axios should be described in the "Given" section.  The "When" section should describe the thing the test does, not what we're hoping it returns.  For example' `Given /system_status is mocked to return CALIBRATED as the status and the current status is CALIBRATING, When ping_system_status is called, Then ...`